themes/theme-b2b/src/tokens.ts

```diff
@@ -32,7 +32,7 @@ export const colors = {
     primary: {
       DEFAULT: brand.secondary[700],
     },
-
+    blue: '#3ab3d5',
```

Comment:
hm.. should we have a blue color? I'm wondering if we could have a better name for it. In core we used dark primary....

Reply:
maybe `link` ? or we make it a lighter gray and name it `primary-light`?

Reply:
hmmm , for me i think `link` is better since this color isn't one of `brand` colors 
what do you think ?!

Reply:
yes. I would name it link

Reply:
now switched to `link`